### PR TITLE
Serialization Fix for Dictionaries With Key as EnumType

### PR DESCRIPTION
**Pull request by saikrishnav**
https://github.com/microsoft/azure-devops-node-api/issues/339

### CODE REVIEW

File: api/Serialization.ts [hunk 1]
```diff
@@ -280,7 +280,6 @@ export module ContractSerializer {
                                     var value = enumType.enumValues[name];
                                     if (name.toLowerCase() === lowerCaseEnumName) {
                                         resultPart = value;
-                                        return resultPart;
```
Comment (damccorm) [inline api/Serialization.ts]: Could we change this to `break;` instead? That way we don't keep spinning on the loop

Comment (saikrishnav) [reply api/Serialization.ts]: makes sense.

**Comment by saikrishnav** [comment]
@damccorm  - Let me know if anything else is required.

**Comment by damccorm** [comment]
LGTM

**Comment by puagarwa** [comment]
#305

**Pull request merged**